packages/components/src/popover/index.tsx

```diff
@@ -139,6 +139,7 @@ const UnforwardedPopover = (
 		shift = false,
 		inline = false,
 		variant,
+		contentStyle,
```

Comment:
Probably need to spin the popover changes out into a separate PR. Or look at alternative options for popover styles inside the canvas.

Reply:
Yep, that's correct, it won't work because of the iframe. Two of the the options are to use inline styles or render a stylesheet. 

Another would be to render the `BlockPopoverCover` inside `<div>` and use that div to reset the margin, but there'd still be the popover z-index to unset. Maybe though the components team would be more open to a `noZIndex` prop or something then though. I'll find out.